doc/build-netbsd.md

```diff
@@ -0,0 +1,81 @@
+NetBSD build guide
+======================
+(updated for NetBSD 8.0)
+
+This guide describes how to build dashd and command-line utilities on NetBSD.
+
+This guide does not contain instructions for building the GUI.
+
+**This guide has not been tested for building Dash Core and is expected to fail due to missing `bls_dash` and `backtrace`. Please report your results; contributions welcome.**
```

Comment:
It's a bit awkward to include instructions that do not work... Why do we want these PRs to be backported in the first place if instructions are useless?

Reply:
Well, I don't think they're useless, they're just not complete and noone here seems invested in completing them. I think there isn't much harm in backporting them